content/tutorials/xgboost_deployment.md

```diff
@@ -39,10 +39,10 @@ watchlist = [(dtrain, 'train')]
 num_round = 2
 bst = xgb.train(param, dtrain, num_round, watchlist)
 ```
-We also must define our predict function for our model. In order to deploy the XGBoost model to Clipper, we define a closure that references the XGBoost model. Clipper's model deployer will automatically track down everything that is referenced in the closure (including the XGBoost model), and include it in the model container.
+We also must define our predict function for our model. In order to deploy the XGBoost model to Clipper, we define a closure that references the XGBoost model. Clipper's model deployer will automatically track down everything that is referenced in the closure (including the XGBoost model), and include it in the model container. We loop through every element in the batch to account for Clipper's adaptive batching, which adds duplicates of requests to batches to figure out the optimal batch size for a model container.
```

Comment:
- which "may" add
- link to adaptive batching: https://docs.google.com/presentation/d/12FLuE6HRnindJOxK_iy7pf8piC6SKR3KwDy-eNP0vlg/edit?usp=sharing